app/controllers/users/two_factor_authentication_controller.rb

```diff
@@ -212,19 +212,19 @@ def track_events(otp_delivery_preference:)
       )
 
       if UserSessionContext.authentication_context?(context)
-        irs_attempts_api_tracker.mfa_phone_verification_otp_sent(
+        irs_attempts_api_tracker.mfa_verify_phone(
           reauthentication: false,
           phone_number: parsed_phone.e164,
           success: @telephony_result.success?,
         )
       elsif UserSessionContext.reauthentication_context?(context)
-        irs_attempts_api_tracker.mfa_phone_verification_otp_sent(
+        irs_attempts_api_tracker.mfa_verify_phone(
```

Comment:
I might lightly nitpick the name changes in this file&mdash;I like the brevity, but I worry the "OTP sent" part gets lost. IOW, I would naively assume that `mfa_verify_phone` is sent when the user has verified their phone number, but it's actually sent when they are sent an SMS OTP.

What about something like `mfa_sms_verify_otp_sent` and `mfa_sms_enroll_otp_sent`? Just spitballing.

Reply:
👍 Point taken - although I'm going with `mfa_verify_phone_otp_sent` since it could be either sms or voice, I believe.